app/views/customers/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @customer.name %></h1>
+
+<h2> Items </h2>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li>
```

Comment:
This li element is where you can add your css ids so you can do targeted testing with a within block.